scripts/veloxbench/veloxbench/cpp_micro_benchmarks.py

```diff
@@ -0,0 +1,121 @@
+import copy
+import json
+import os
+import tempfile
+
+import conbench.runner
+
+from veloxbench import _benchmark
+
+RUN_OPTIONS = {
+    "iterations": {
+        "default": None,
+        "type": int,
+        "help": "Number of iterations of each benchmark.",
+    },
+}
+
+
+# TODO: what other COMMON_OPTIONS?
+# https://github.com/ursacomputing/benchmarks/blob/033eee0951adbf41931a2de95caccbac887da6ff/benchmarks/cpp_micro_benchmarks.py#L18-L59
+# TODO: the suite filter won't quite work yet either
+COMMON_OPTIONS = {
+    "suite-filter": {
+        "default": None,
+        "type": str,
+        "help": "Regex filtering benchmark suites.",
+    },
+}
+
+
+def get_run_command(filename, options):
+    # TODO: we'll probably want to unify these runner scripts
+    # Must run `make benchmarks-basic-build` before this
+    # TODO: deal with the paths better (especially to benchmarks/basic)
+    command = [
+        "../../../scripts/benchmark-runner.py",
+        "run",
+        "--path",
+        "../../../_build/release/velox/benchmarks/basic/",
+        "--dump-path",
+        filename,
+    ]
+
+    # TODO: extend cpp micro benchmarks to allow for iterations
+    iterations = options.get("iterations", None)
+    if iterations:
+        raise NotImplementedError()
```

Comment:
We might not need this for microbenchmarks (and we might actually want tuning parameters like how long to let them run for and look at iterations per second)

Reply:
Yeah, the benchmark-runner script already hard codes parameters around iterations/timing, but we could make sure they are properly plumbed in the future.